docs/source/guide/scim_setup.md

```diff
@@ -0,0 +1,126 @@
+
+Using SCIM provider, you can manage access to Label Studio Enterprise workspaces, grant roles to individual users and groups.
+
+## Requirements
+
+SCIM interacts with your SSO integration. 
+
+!!! note
+Okta or similar SSO providers have SCIM integration based on SSO.
+
+
+!!! attention "important"
+If you do not have SSO set up yet, then follow [Set up SSO](auth_setup.md).
+
+Label Studio Enterprise uses SCIM Version 2.0 standard. As an example, this page uses integration with [Okta](https://www.okta.com/integrate/). However, Label Studio Enterprise follows [SCIM RFC 5741](https://datatracker.ietf.org/doc/html/rfc7644#section-3.2) and can be integrated with any access management services that support the standard.
+
+## Set up SCIM integration
+
+<i>Check this video tutorial about SCIM and Okta setup</i>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/MA3de3gu18A" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+To manage access to Label Studio Enterprise, add the application to your SCIM provider (Okta). Okta uses OAuth token to interact with REST API endpoints of the application to provision and deprovision access.
+
+### Add Label Studio Enterprise (if not complete)
+
+1. Navigate to **Applications → Applications** in Okta. Click  **Create App Integration**. 
+2. Select **SAML 2.0**. Fill App name (for example, _Label Studio Enterprise_).
+3. On the next step **Configure SAML** set up SAML integration following the instructions to [Set up SSO guide](auth_setup.md).
+4. Make sure Label Studio Enterprise appears in the list of active applications.
+
+### Enable SCIM provisioning
+
+1. Navigate to **Applications → Applications** in Okta. 
+2. Select **Label Studio Enterprise**.
+3. Go to **General** tab and check **Enable SCIM provisioning**.
+4. Switch to **Provisioning** tab. 
+5. Select **Integration** in the left menu. 
+6. Click **Edit** in the right corner.
+
+Fill in the fields:
+ - **SCIM connector base URL**: https://{LABEL_STUDIO_BASE_URL}/scim/v2/ where `{LABEL_STUDIO_BASE_URL}` is the base URL of your Label Studio Enterprise instance.
+ - **Unique identifier field for users**: Leave `userName`. Label Studio Enterprise uses email as user identifier in this field.
+ - **Supported provisioning actions**: Select the following items:
+   - Import New Users and Profile Updates
+   - Push New Users
+   - Push Profile Updates
+   - Push Groups
+ - **HTTP Header → Authorization**: Put the OAuth token from the **Settings** page.
+
+### SCIM settings and application triggers
+
+1. On the application page navigate to **Provisioning** tab and select **To App** in the left menu. Click **Edit** in the right corner.
+2. Enable the following items:
+   - Create Users
+   - Update User Attributes
+   - Deactivate Users
+   - Sync Password
+
```

Comment:
Let's maintain two empty lines before ## (H2 headers) and one empty line before ### (H3 headers)